DFC.ServiceTaxonomy.TestSuite/PageObjects/AddContentItemBase.cs

```diff
@@ -111,8 +126,12 @@ public AddContentItemBase EnterText(string field, string value, By locator)
             IWebElement item = null;
             try
             {
+                if (_htmlView)
+                {
+                   item = _scenarioContext.GetWebDriver().FindElement(By.CssSelector(".trumbowyg-viewHTML-button > svg"));
```

Comment:
We'll have to look at adding an id to the html field at some point, for content items with more than 1 html field.

Reply:
yes agree.  It can be modified at a later date to grab a collection of editor elements and reference the required one by index